package.json

```diff
@@ -53,6 +53,12 @@
   "jest": {
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
-    ]
+    ],
+    "eslint-plugin-jsx-a11y": "^6.2.1",
```

Comment:
These packages should not be listed here. They should either be in `dependecies` or `devDependencies`

Reply:
You haven't fixed this as well

Reply:
You haven't fixed this as well